global/prometheus-alertmanager/slack.tmpl

```diff
@@ -1,3 +1,4 @@
+{{ define "slack.sapcc.color" }}{{ if .Alerts.Firing | and .Alerts.Resolved }}warning{{ else if .Alerts.Firing }}danger{{ else }}good{{ end }}{{ end }}
```

Comment:
Not sure I understand the syntax 1st if part: Should'nt it be `{{ if and .Alerts.Firing .Alerts.Resolved) }}warning ...`? 
Besides LGTM

Reply:
https://golang.org/pkg/text/template/#hdr-Pipelines

> A pipeline may be "chained" by separating a sequence of commands with pipeline characters '|'. In a chained pipeline, the result of each command is passed as the last argument of the following command.

Reply:
It looks a little weird but is perfectly valid and allows to read it like `condition1 and condition2` which I like better then the reverse polish notation `and condition1 condition2`